PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetHUtils.h

```diff
@@ -163,7 +171,7 @@ class FlowVectorCorrections {
   static void AddRawFMD(AliAnalysisTaskFlowVectorCorrections *task, AliQnCorrectionsManager* QnManager, AliQnCorrectionsVarManagerTask::Variables varForEventMultiplicity);
   static void DefineHistograms(AliQnCorrectionsManager* QnManager, AliQnCorrectionsHistos* histos, TString histClass);
 };
-
+*/
```

Comment:
Why are you commenting this out?

Reply:
Sorry for making this mistake again. I uncommented it. The reason I comment it out when testing locally is because if I do not, then I get the following errors when compiling it:

In file included from ./AliAnalysisTaskEmcalJetHPerformance.h:23:
/home/alidock/Raymond_Code/Raymond_Latest_Code/AliAnalysisTaskEmcalJetHUtils_v2.h:163:7: error: redefinition of 'FlowVectorCorrections'
class FlowVectorCorrections {
      ^
/home/alidock/alice/sw/slc7_x86-64/AliPhysics/master_ROOT6-local2/include/AliAnalysisTaskEmcalJetHUtils.h:155:7: note: previous definition is here
class FlowVectorCorrections {
 
...

fatal error: too many errors emitted, stopping now [-ferror-limit=]
Error in <TInterpreter::AutoParse>: Error parsing payload code for class PWGJE::EMCALJetTasks::AliAnalysisTaskEmcalJetHdEdxCorrelations with content:

Reply:
It's because you have two copies and didn't change the name of the class in your new copy